docs/sources/helm-charts/mimir-distributed/configure/configure-gem-federation-frontend.md

```diff
@@ -0,0 +1,155 @@
+2. Provision an access token with `metrics:read` scope for each cluster that you want to query. For more information, refer to [Set up a GEM tenant](https://grafana.com/docs/enterprise-metrics/<GEM_VERSION>/set-up-gem-tenant).
+
+## Deploy the GEM Federation-Frontend
+
+1. Create a Kubernetes `Secret` named `gem-tokens` with the GEM access tokens for each of the remote GEM clusters. This `Secret` will be later used in the Helm values file. Replace `TOKEN1` and `TOKEN2` with the access tokens for the remote GEM clusters.
+
+   ```yaml
+   apiVersion: v1
+   kind: Secret
+   metadata:
+     name: gem-tokens
+   data:
+     CLUSTER_LOCAL_GEM_PASSWORD: TOKEN1
+     CLUSTER_REMOTE_GEM_PASSWORD: TOKEN2
+   ```
+
+2. Apply the secret to your cluster in the `federation-frontend-demo` namespace with the `kubectl` command:
+
+   ```bash
+   kubectl -n federation-frontend-demo apply -f mysecret.yaml
+   ```
+
+3. Create a values file named `federation-frontend.yaml` with the following content.
+
+   Replace `http://gem-query-frontend.monitoring.svc.cluster.local:8080/prometheus` and `https://gem.monitoring.acme.local/prometheus` with the URLs of the GEM clusters you want to query. Replace `tenant-1` and `tenant-2` with the tenant IDs of the remote GEM clusters.
+
+   Note that the resource settings shown below are examples that should be sufficient for small deployments. Adjust the values based on your specific requirements and load:
+
+   ```yaml
+   # Enable enterprise features
+   enterprise:
+     enabled: true
+
+   # Enable and configure federation-frontend
+   federation_frontend:
+     enabled: true
+     replicas: 2
+     resources:
+       requests:
+         cpu: 100m
+         memory: 128Mi
+       limits:
+         cpu: 1
+         memory: 256Mi
+     extraEnvFrom:
+       - secretRef:
+           name: gem-tokens
+
+   # Configure the remote GEM clusters to query.
+   mimir:
+     structuredConfig:
+       auth:
+         type: trust
+       # The federation-frontend doesn't handle authentication and authorization. Disabling multitenancy means the federation-frontend will not require the X-Scope-OrgID header.
+       # With disabled multitenant the federation-frontend will use the authn/z material from the proxy_targets configuration.
+       # For a complete list of configuration options, refer to the configuration reference at https://grafana.com/docs/enterprise-metrics/<GEM_VERSION>/config/reference/#federation.
+       multitenancy_enabled: false
+       federation:
+         proxy_targets:
+           - name: "cluster-local-gem"
+             url: "http://gem-query-frontend.monitoring.svc.cluster.local:8080/prometheus"
+             basic_auth:
+               username: tenant-1
+               password: "${CLUSTER_LOCAL_GEM_PASSWORD}"
+
+           - name: "cluster-remote-gem"
```

Comment:
Perhaps cluster 1 and cluster 2 would be better names to use here? Might be confusing to refer to one as local and one as remote given there's no requirement about where the clusters are.

Reply:
👍 done in https://github.com/grafana/mimir/pull/9735/commits/909a59625b30a74edfea842ce06e062d1e22347a